static/js/tools/map/chart.tsx

```diff
@@ -433,23 +435,19 @@ const getTooltipHtml = (
     !_.isEmpty(metadata.popDate) &&
     !metadata.statVarDate.includes(metadata.popDate) &&
     !metadata.popDate.includes(metadata.statVarDate);
-  let statVarTitle = statVar.info.title ? statVar.info.title : statVar.dcid;
-  if (showPopDateMessage) {
-    statVarTitle += "<sup>1</sup>";
-  }
   if (!hasValue || !(place.dcid in metadataMapping)) {
-    return titleHtml + `${statVarTitle}: <wbr>${value}<br />`;
+    return `${titleHtml}: <wbr>${value}<br />`;
   }
   if (!_.isEmpty(metadata.errorMessage)) {
-    return titleHtml + `${statVarTitle}: <wbr>${metadata.errorMessage}<br />`;
+    return `${titleHtml}: <wbr>${metadata.errorMessage}<br />`;
   }
-  const popDateHtml = showPopDateMessage
-    ? `<sup>1</sup> Uses population data from: <wbr>${metadata.popDate}`
+  const footer = showPopDateMessage
+    ? `<footer><sup>1</sup> Uses population data from: <wbr>${metadata.popDate}</footer>`
     : "";
   const html =
-    titleHtml +
-    `${statVarTitle} (${metadata.statVarDate}): <wbr><b>${value}</b><br />` +
-    `<footer>${popDateHtml}</footer>`;
+    `${titleHtml} (${metadata.statVarDate}): <wbr><b>${value}</b>${
+      showPopDateMessage ? "<sup>1</sup>" : ""
+    }<br />` + footer;
```

Comment:
just pointing it out (no need to update your code), but in this case - you could also just always add ${footer} to the format string.